packages/cycle-scripts/template/public/index.html

```diff
@@ -1,15 +1,12 @@
 <!DOCTYPE html>
```

Comment:
no need to use an .ejs with the TMLWebpackPlugin, you can just use an html file and inject into it, no?